docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+                            },
+                            "NGINX_CONFIG_ENV_VAR_LIST": {
+                                    "value": "NGINX_DEFAULT_PORT,BLOB_UPLOAD_ROUTE_ADDRESS,CONNECTED_ACR_ROUTE_ADDRESS,IOTEDGE_PARENTHOSTNAME,DOCKER_REQUEST_ROUTE_ADDRESS"
+                            },
+                            "DOCKER_REQUEST_ROUTE_ADDRESS": {
+                                "value": "registry:5000"
+                            },
+                            "BLOB_UPLOAD_ROUTE_ADDRESS": {
+                                "value": "AzureBlobStorageonIoTEdge:11002"
+                            }
+                        },
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "startupOrder": 3
+                    }
+                },
+                "runtime": {
+                    "settings": {
+                        "minDockerVersion": "v1.25",
+                        "registryCredentials": {
+                            "tsmregistry": {
+                                "address": "10.16.7.4",
+                                "password": "$$$0meCoMPL3xP4$$W0rd001!@#$$",
+                                "username": "nestedcrimagepulltoken"
+                            }
+                        }
+                    },
+                    "type": "docker"
+                },
+                "schemaVersion": "1.1",
+                "systemModules": {
+                    "edgeAgent": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-agent:1.0",
+                            "createOptions": ""
+                        },
+                        "type": "docker",
+                        "env": {
+                            "SendRuntimeQualityTelemetry": {
+                                "value": "false"
+                            }
+                        }
+                    },
+                    "edgeHub": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-hub:1.0",
+                            "createOptions": "{\"HostConfig\":{\"PortBindings\":{\"443/tcp\":[{\"HostPort\":\"443\"}],\"5671/tcp\":[{\"HostPort\":\"5671\"}],\"8883/tcp\":[{\"HostPort\":\"8883\"}]}}}"
+                        },
+                        "type": "docker",
+                        "status": "running",
+                        "restartPolicy": "always"
+                    }
+                }
+            }
+        },
+        "$edgeHub": {
+            "properties.desired": {
+                "routes": {
+                    "route": "FROM /messages/* INTO $upstream"
+                },
+                "schemaVersion": "1.1",
+                "storeAndForwardConfiguration": {
+                    "timeToLiveSecs": 7200
+                }
+            }
+        }
+    }
+}
+```
+
+Use the information from the previous sections to update the relevant JSON values.
```

Comment:
@toddysm can we help users by explicitly clarifying which sections need to be updated? It may not be immediately obvious since the sample JSON has all real values and no placeholders. Just want to make sure they do not miss any sections.

Reply:
I added few bullet points below. Let me know if it works.